sechub-adapter-pds/src/main/java/com/daimler/sechub/adapter/pds/PDSWebScanConfigImpl.java

```diff
@@ -82,6 +82,22 @@ public final PDSWebScanConfigBuilder setJobParameters(Map<String, String> jobPar
         @Override
         protected void customBuild(PDSWebScanConfigImpl config) {
             jobParameters.put(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_TARGET_TYPE, config.getTargetType());
+            
+            // to get the base URl from the URL
+            // https://example.org/abc -> https://example.org
+            
+            try
+            {            
+            	URL url = new URL(PARAM_KEY_PDS_SCAN_TARGET_URL);
+				String path = url.getFile().substring(0, url.getFile().lastIndexOf('/'));
+				String baseUrl = url.getProtocol() + "://" + url.getHost() + path;
+				PARAM_KEY_PDS_SCAN_TARGET_URL = baseUrl;
+           	}
+           	catch(Exception e)
+ 			{
+ 				throw new IllegalArgumentException(PARAM_KEY_PDS_SCAN_TARGET_URL + " is not a valid URL ",e);
+ 			}
+ 			
             jobParameters.put(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_TARGET_URL, config.getTargetAsString());
```

Comment:
Just for information: We put here into jobParameters for the key `PARAM_KEY_PDS_SCAN_TARGET_URL` the config target string. The framework will automatically inject this one at PDS side as an ENV entry